firmware/brake/kia_soul_ps/brake_control_module.ino

```diff
@@ -512,8 +517,8 @@ void turn_off_all_brake_solenoids( )
 // *****************************************************
 void turn_on_brake_actuator_solenoids( int16_t scaler )
```

Comment:
Please fix the spelling of scalar, if that's what it's supposed to be.  If this is something that's supposed to 'scale' then it's probably ok, but should be renamed to something more descriptive.